apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -973,6 +981,24 @@
     timestamp: now - (60000000 * 20)
   });
 
+  messagesDb.messages.push({
+    type: 'mms',
+    id: messagesDb.id++,
+    threadId: 15,
+    delivery: 'received',
+    deliveryInfo: [{
+      deliveryStatus: 'success',
+      deliveryTimestamp: 0,
+      readStatus: 'not-applicable',
+      readTimestamp: 0,
+      receiver: null
+    }],
+    sender: '+33623456789',
+    receivers: ['+33623456789','+33612345678'],
+    timestamp: now - (60000000 * 30),
+    sentTimestamp: now - (60000000 * 30)
```

Comment:
then the message will be in error because a MMS should either have an attachment or a subject, but it's still good enough to open the report